webofneeds/won-owner-webapp/src/main/java/won/owner/web/WonOwnerMailSender.java

```diff
@@ -259,21 +168,61 @@ public void sendConnectNotificationHtmlMessage(String toEmail, String localNeed,
 
   }
 
-  public void sendHintNotificationMessageHtml(String toEmail, String localNeed, String remoteNeed) {
+  public void sendCloseNotificationHtmlMessage(String toEmail, String localNeed, String
+    remoteNeed, String localConnection, String textMsg) {
+
+    String ownerAppLink = uriService.getOwnerProtocolOwnerURI().toString();
+    Dataset needDataset =  linkedDataSource.getDataForResource(URI.create(remoteNeed));
+    String remoteNeedTitle = WonRdfUtils.NeedUtils.getNeedTitle(needDataset, URI.create(remoteNeed));
+    String linkLocalNeed = ownerAppLink + OWNER_LOCAL_NEED_LINK + localNeed;
+    String linkRemoteNeed = uriService.getOwnerProtocolOwnerURI() + OWNER_REMOTE_NEED_LINK + remoteNeed;
+    String linkConnection = ownerAppLink + String.format(OWNER_CONNECTION_LINK,
+                                                         localNeed, localConnection, ConnectionState.CLOSED.getURI()
+                                                                                                                     .toString());
+    // TODO implement login in context functionality for the linked from the email owner interface
+
+    String theyWrote = "</p>";
+    if (textMsg != null) {
+      theyWrote = "They wrote:" +
+        "</p>" +
+        "<p style=\"border-left:thick solid #808080;\">" +
+        "   <span style=\"margin-left:5px;color:#808080;\">" + textMsg + "</span>" +
+        "</p>";
+    }
+
+    String body =
+      "<div>" +
+        NOTIFICATION_START_HTML +
+        "<p>The owner of '<a href=\"" + linkRemoteNeed + "\">" + remoteNeedTitle + "</a>'" +
+        "   closed the connection to <a href=\"" + linkLocalNeed + "\">you</a>. " +
+        theyWrote +
+        "<p><a href=\"" + linkConnection + "\">[Click here to view the conversation]</a></p>" +
+        NOTIFICATION_END_HTML +
+        "</div>";
+
+
+    logger.debug("sending " + SUBJECT_CONNECT + " to " + toEmail);
+
+    this.wonMailSender.sendHtmlMessage(toEmail, SUBJECT_CONNECT, body);
```

Comment:
this should be another subject for the closing of messages not "new conversation request" (SUBJECT_CONNECT)